R/build-articles.R

```diff
@@ -398,12 +398,12 @@ data_articles_index_section <- function(section, pkg) {
     name = section_vignettes$name,
     path = path_rel(section_vignettes$file_out, "articles"),
     title = section_vignettes$title,
-    description = lapply(section_vignettes$description, markdown_text_block, pkg = pkg),
```

Comment:
pkg wasn't used. It's only used in `markdown_text_inline` for an error message.

Reply:
In retrospect this refactoring could have happened elsewhere but I thought I could avoid using pkg for data_news(), turns out it'd wouldn't be clearer. :sweat_smile:

Reply:
I pulled this all out in to a separate commit to make it easier to review this PR.